convert/README.md

```diff
@@ -0,0 +1,185 @@
+# Domino CDK to Terraform migration
+
+Domino is deprecating this CDK repository, and switching all future EKS
+provisioning work to our Terraform module, located here:
+
+https://github.com/dominodatalab/terraform-aws-eks
+
+The scripts in this directory will help you convert the management your
+CDK-provisioned Domino infrastructure to the above Terraform module.
+
+Your EKS cluster, EFS store, and S3 buckets will all remain in-tact.
+The original ASGs and any bastion used for the install will be replaced
+by ones generated in Terraform.
+
+## Contents
+
+* convert.py - The core migration script
+* README.md - This file
+* terraform/ - Terraform to take over CDK infrastructure
+* cloudformation-only/ - Terraform to create a special IAM role to remove the CDK CloudFormation stack
+
+## Migration steps
+
+To convert the CDK stack to terraform, we will...
+
+1. Create input variables into Terraform
+2. Generate a set of Terraform import commands to import various AWS resources into the new Terraform
+3. Initialize the Terraform directory and import the resources
+4. Evaluate the Terraform plan
+5. Apply the Terraform plan
+6. Scale down the old auto-scaling groups
+7. Validate Domino functionality
+8. Delete unneeded AWS resources
+9. Delete the old CloudFormation stack
+
+### convert.py usage
+
+    usage: convert.py [-h] {create-tfvars,resource-map,get-imports,delete-stack,print-stack} ...
+    
+    terraform eks module importer
+    
+    options:
+      -h, --help            show this help message and exit
+    
+    commands:
+      {create-tfvars,resource-map,get-imports,delete-stack,print-stack}
+	create-tfvars       Generate tfvars
+	resource-map        Create resource map for customization/debugging of get-imports command (optional step)
+	get-imports         Get terraform import commands
+	delete-stack        Get commands to delete old stack
+	print-stack         Print CDK stack resources
+
+### Create terraform variables
+
+First, we must create the variables going into our Terraform config.
+
+The following command will look at the current CDK stack and autopropagate the Terraform variables:
+
+    ./convert.py create-tfvars --region us-east-1 --stack-name mystack --ssh-key-path /path/to/key.pem > terraform/terraform.tfvars
```

Comment:
This needs to output to `terraform.tfvars.json` instead of `terraform.tfvars`. The content is valid json, not an HCL or key-value format, so when it is used later in `imports.sh` it throws an error.

```
+ ./imports.sh
+ terraform import module.domino_eks.module.storage.aws_efs_file_system.eks fs-0172b729c2ee51a56
╷
│ Error: Argument or block definition required
│
│   on terraform.tfvars line 1:
│    1: {"deploy_id": ...
```
Using `tfvars.json` clears that issue.

Reply:
Doh, yeah. I've made that mistake working with it manually as well, so not suprising... fixed.